src/js/components/TextInput/StyledTextInput.js

```diff
@@ -9,9 +10,25 @@ import {
 } from '../../utils';
 import { defaultProps } from '../../default-props';
 
+const getPlainStyle = plain => {
+  if (plain === 'full') {
+    return css`
+      outline: none;
+      border: none;
+      padding: 0;
+    `;
```

Comment:
we should leverage the existing `plainInputStyle` instead of specifying a separate css.

Reply:
Updated to use `plainInputStyle`
```javascript
return css`
      ${plainInputStyle} padding: 0;
    `;
```